### PR TITLE
feat: add `math/base/special/cscd`

### LINKED ISSUE #39: Implement `cscd`
Compute the cosecant of `x` where `x` is in degrees. `@stdlib/math/base/special/cscd`.

### PR DESCRIPTION
**Pull request by saisrikardumpeti**
Resolves #39 

## Description

> What is the purpose of this pull request?

This pull request:

-   Adds functionality to compute `cscd`  of any input (in degree).

| degree    | secd |
| -------- | ------- |
| 0  | Infinity |
| 30 |  2  |
| 45 | 1.414    |
| 60 | ~1.15 |
| 90 |1.0  |

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #39
-   fixes #

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/math/base/special/cscd/lib/main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+* var v = cscd( 0 );
+* // returns Infinity
+*
+* @example
+* var v = cscd( NaN );
+* // returns NaN
+*/
+function cscd( x ) {
+	var rad = deg2rad(x);
+	return 1 / sin(rad);
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/cscd/lib/main.js:61]:
```suggestion
	return 1.0 / sin(rad);
```

Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/cscd/lib/main.js]:
```suggestion
	var rad = deg2rad( x );
	return 1.0 / sin( rad );
```

File: lib/node_modules/@stdlib/math/base/special/cscd/package.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@stdlib/math/base/special/cscd",
+  "version": "0.0.0",
+  "description": "Compute the cosecant of degree.",
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/math/base/special/cscd/package.json]:
```suggestion
  "description": "Compute the cosecant of a degree.",
```

**Review by Planeshifter — APPROVED** [review]
LGTM! Thanks!

**Comment by saisrikardumpeti** [comment]
i have to update 4 more PRs i'll let you know.

**Pull request merged**